integration-test/tests/cms/footer.test.ts

```diff
@@ -1,14 +1,29 @@
 import {expect, test} from '@playwright/test';
 import { footerExpectedInformation } from '../../utils/datafactory/footer.data';
+import { validateSchema } from '../../utils/helpers/schema.validation';
+import { footerSchema } from '../../utils/datafactory/schemas/footer.schema';
+
 
 test('GET /api/cms/v1/footer returns correct footer data', async ({request}) => {
   const response = await request.get(`/api/cms/v1/footer`);
 
+  // response status validation
   expect(response.status()).toBe(200);
 
   const body = await response.json();
 
-  const expectedResponse = footerExpectedInformation;
+  // schema validation
```

Comment:
I think would be better to create on test per topic, for example one for schema and one for status code, because usually is a good practice to split and have one only reason to the test failure, so you don't need to add inline comment but you can have 

test footer success
test footer schema 
test footer response

Reply:
I can work on this example in another pr

Reply:
In this case I think that if API fails with status code different than 200, I should not even try to check other things like schema and body for not successful call (it will fail anyway). 
That's why we first check status code, then schema and then if necessary - body.